api/composer.json

```diff
@@ -38,7 +38,8 @@
     "larastan/larastan": "^3.0.2",
     "laravel/pint": "^1.11",
     "mockery/mockery": "^1.4.4",
-    "phpunit/phpunit": "^10.0",
+    "pestphp/pest": "^3.0",
+    "phpunit/phpunit": "^11.0",
```

Comment:
v11 will end bugfix support in a few months and is only compatible with PHP 8.2 (not sure if this actually matters since we are using PHP 8.4 and v12 only says it supports up to 8.3 🤷).

https://phpunit.de/supported-versions.html